app/server/postgraphile/uploadFieldPlugin.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable */
```

Comment:
This is copy-pasted from CIF with no linting errors, so I'm assuming it's fine to disable here